install_pim/installation_archive.rst.inc

```diff
@@ -5,29 +5,53 @@ Extracting the archive
     :linenos:
 
     $ mkdir -p /path/to/installation
-    $ tar -xvzf pim-community-standard-v1.8-latest-icecat.tar.gz -C /path/to/installation/
+    $ tar -xvzf pim-community-standard-v2.0-latest-icecat.tar.gz -C /path/to/installation/
 
 .. note::
     Replace */path/to/installation* by the path to the directory where you want to install the PIM.
 
 .. note::
-    * For Community Edition, replace *pim-community-standard-v1.8-latest-icecat.tar.gz* by the location and the name
+    * For Community Edition, replace *pim-community-standard-v2.0-latest-icecat.tar.gz* by the location and the name
       of the archive you have downloaded from https://www.akeneo.com/download
-    * For Enterprise Edition, replace *pim-community-standard-v1.8-latest-icecat.tar.gz* by the location and the name
+    * For Enterprise Edition, replace *pim-community-standard-v2.0-latest-icecat.tar.gz* by the location and the name
       of the archive you have downloaded from `Partner portal <https://partners.akeneo.com/login>`_
 
 .. note::
     The PIM will be extracted in the folder /path/to/installation/pim-community-standard.
 
 Initializing Akeneo
 -------------------
+
 .. code-block:: bash
     :linenos:
 
     $ cd /path/to/installation/pim-community-standard
-    $ ../composer.phar install --optimize-autoloader --prefer-dist  # optional for community edition
-    $ php app/console cache:clear --env=prod
-    $ php app/console pim:install --env=prod
+
+Dependencies,
+
+.. code-block:: bash
+    :linenos:
+
+    $ ../composer.phar install --optimize-autoloader --prefer-dist memory_limit=3G
```

Comment:
```diff
- composer.phar install --optimize-autoloader --prefer-dist memory_limit=3G
+ php -d -d memory_limit=3G composer.phar install --optimize-autoloader --prefer-dist
```

Reply:
nooo :'( too fast :/

Reply:
I changed it in upcoming PR https://github.com/akeneo/pim-docs/pull/673

Thank you mate :wink: